main.go

```diff
@@ -90,11 +92,12 @@ func NewGame(g *Game) {
 
 	//Load sprites
 	g.Sprites = make(map[SpriteType]*SpriteSheet, 2)
-	g.Sprites[spritePlayer] = loadSprite("player")
-	g.Sprites[spriteZombie] = loadSprite("zombie")
+	g.Sprites[spritePlayer] = loadSprite("eZcort mission_Player Character")
+	g.Sprites[spriteZombie] = loadSprite("eZcort mission_Zombie")
```

Comment:
Meh it would be cooler if we renamed the PNGs to nice short names.

Reply:
I asked Acros to use simple names, then we don't need to hassle with them always during a sprite update